src/Symfony/Component/Security/Http/AccessToken/AccessTokenHandlerInterface.php

```diff
@@ -24,5 +25,5 @@ interface AccessTokenHandlerInterface
     /**
      * @throws AuthenticationException
      */
-    public function getUserIdentifierFrom(#[\SensitiveParameter] string $accessToken): string;
+    public function getUserIdentifierFrom(#[\SensitiveParameter] string $accessToken): UserBadge|string;
```

Comment:
```suggestion
    public function getUserFrom(#[\SensitiveParameter] string $accessToken): UserBadge|string;
```

This is more explicit, and in sync with the default use case when using OIDC or any other system relying on JWT.

Reply:
The method does not return a User instance though. I suggest`getUserBadgeFrom(string $accessToken)` with phpdoc specifying that it may return a string-ish user identifier instead.

Reply:
Changed it to `getUserBadgeFrom`.